baseapp/baseapp_core/graphql/object_types.py

```diff
@@ -28,3 +28,7 @@ def __init_subclass_with_meta__(cls, **kwargs):
 
     def resolve_pk(self, info):
         return self.pk
+
+    @classmethod
```

Comment:
@luciobj I guess we don't need it anymore since we are removing the django optimizer